aodndata/moorings/products_handler.py

```diff
@@ -310,9 +344,15 @@ def preprocess(self):
 
         if 'aggregated' in self.products_to_create:
             self._make_aggregated_timeseries()
```

Comment:
Up to you, but it's I was expecting a new class of MooringProductHandler to be created for each type of product - after all they are all different classes. 
The pros is that this would clearly avoid any possible side effects and reduce logic handling of nested dependencies.

Reply:
I think it makes sense to do it have the products generated by the same process, because
1. whenever there's new data for a site, we'll want to re-generate _all_ the products for that site;
1. multiple products are generated from the same set of input files; and
1. the gridded product takes the hourly as an input.